kb/section_subjdomain_of_natural_language_texts/classification.scs

```diff
@@ -0,0 +1,201 @@
+lexeme
```

Comment:
This file is unnecessary, move in `.scsi`, maybe. In entities directory we store entities only. Need to fix in next PRs.